pkg/monitortests/cloud/azure/metrics/monitortest.go

```diff
@@ -171,8 +172,32 @@ func (w *azureMetricsCollector) StartCollection(ctx context.Context, adminRESTCo
 		return w.notSupportedReason
 	}
 
-	// Only collect if we are on azure
+	// Create oc client earlier to use for hypershift check
 	oc := exutil.NewCLI("cloudmetrics").AsAdmin()
+	var isAROHCPcluster bool
+	isHypershift, _ := exutil.IsHypershift(ctx, oc.AdminConfigClient())
+	if isHypershift {
+		_, hcpNamespace, err := exutil.GetHypershiftManagementClusterConfigAndNamespace()
+		if err != nil {
+			logrus.WithError(err).Error("failed to get hypershift management cluster config and namespace")
+		}
+		logrus.Infof("hcpNamespace is: %s", hcpNamespace)
+
+		// For Hypershift, only skip if it's specifically ARO HCP
+		if isAROHCPcluster, err := isAROHCP(ctx, hcpNamespace, oc.KubeClient()); err != nil {
```

Comment:
Maybe we can replace to use `exutil.NewHypershiftManagementCLI("cloudmetrics")` with `isAROHCP` ?